contents/docs/self-host/configure/3rd-party-clickhouse.md

```diff
@@ -1,9 +1,13 @@
 ---
-title: Deploying ClickHouse using Altinity.Cloud
+title: Deploying ClickHouse using a 3rd party service
 sidebar: Docs
 showTitle: true
 ---
 
+We are gradually adding support for 3rd parties to manage the individual services such as ClickHouse needed to run PostHog. This avoids the need for you to send data to PostHog whilst reducing the management overhead of running the product.
```

Comment:
Note there's no other 3rd party services right now (that I'm aware of) - we can't be adding support for things that don't exist yet.

Given we specifically call out Altinity cloud in the marketplace and it's the only one for now - why not make the relationship explicit?